drape_frontend/drape_engine.cpp

```diff
@@ -994,4 +994,11 @@ void DrapeEngine::UpdateMyPositionRoutingOffset(bool useDefault, int offsetY)
                                   make_unique_dp<UpdateMyPositionRoutingOffsetMessage>(useDefault, offsetY),
                                   MessagePriority::Normal);
 }
+
+void DrapeEngine::SetCustomArrow3d(std::optional<Arrow3dCustomDecl> arrow3dCustomDecl)
+{
+  m_threadCommutator->PostMessage(ThreadsCommutator::ResourceUploadThread,
+                                  make_unique_dp<Arrow3dRecacheMessage>(std::move(arrow3dCustomDecl)),
```

Comment:
```suggestion
                                  std::make_unique_dp<Arrow3dRecacheMessage>(std::move(arrow3dCustomDecl)),
```
We prefer to avoid using namespace std; in the global scope to avoid Unity build failures.

Reply:
make_unique_dp is a drape wrapper for std::make_unique, it should not have std